### PR TITLE
NO JIRA. Corrections in README

### PR DESCRIPTION
**Pull request by janvanmansum**
NO JIRA.

- [x] Solve build errors.

#### When applied it will
* Correct terminology usage. `datasetId` means Fedora-id, e.g., easy-dataset:123, `bagId` means the UUID of a single bag.
* Correct some old boilerplate docs about the installation.
* Updated the style of the command line synopsis.

#### Where should the reviewer @DANS-KNAW/easy start?
The readme, the only file that was updated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,31 +39,25 @@ EXAMPLES
...
-   on the path, e.g. 
-   
-        ln -s /usr/local/easy-transform-metadata-<version>/bin/easy-transform-metadata /usr/bin
-
-
-
-General configuration settings can be set in `cfg/application.properties` and logging can be configured
-in `cfg/logback.xml`. The available settings are explained in comments in aforementioned files.
-
+Currently this project is build only as an RPM package for RHEL7/CentOS7 and later. The RPM will install the binaries to
```
Comment (rvanheest) [inline README.md:42]: @janvanmansum is this also applied in all the other projects? At least it isn't in `easy-module-archetype`, as I just created a new project together with @PaulBoon.

Besides, does this mean that the `bin.xml` can be removed in all projects?

For both points, if they're not yet done or cannot be done immediately, please create one or more issues for them. Thanks!

File: README.md [hunk 2]
```diff
@@ -39,31 +39,25 @@ EXAMPLES
...
 * Maven 3.3.3 or higher
+* RPM
 
 Steps:
-
-        git clone https://github.com/DANS-KNAW/easy-transform-metadata.git
-        cd easy-transform-metadata
-        mvn install
+    
+    #!bash
```
Comment (rvanheest) [inline README.md]: Why is this line necessary?

File: README.md [hunk 3]
```diff
@@ -39,31 +39,25 @@ EXAMPLES
...
+    git clone https://github.com/DANS-KNAW/easy-transform-metadata .git
```
Comment (rvanheest) [inline README.md]:
```suggestion
    git clone https://github.com/DANS-KNAW/easy-transform-metadata.git
```

File: README.md [hunk 4]
```diff
@@ -39,31 +39,25 @@ EXAMPLES
...
+    cd easy-transform-metadata 
+    mvn install
```
Comment (rvanheest) [inline README.md]:
```suggestion
    mvn clean install
```

**Review by rvanheest — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (janvanmansum) [reply README.md]: Oops copied it from `easy-bag-store`, which has been converted to `mkdocs`, where this has meaning. I'll remove it.

File: README.md (hunk 4 above)
Comment (janvanmansum) [reply README.md]: Not strictly necessary if you have just cloned the project, right?

File: README.md (hunk 1 above)
Comment (janvanmansum) [reply README.md:42]: I will update the template README in the archetype.

As for `bin.xml`, it depends on whether we still want to be able to build a `tar.gz` file. Currently, I think building the `tar.gz` is turned off in the build, as we only use the `rpm` anyway. A reason to the `bin.xml` could be to have some minimal support for non-RPM-based systems, in case anybody out there wants to install our software.

File: README.md (hunk 2 above)
Comment (rvanheest) [reply README.md]: no you didn't remove it; lol. You even added it to [`easy-module-archetype`](https://github.com/DANS-KNAW/easy-module-archetype/pull/47/files#diff-68e651711b2b31f340c37ee386107dd7R54)

File: README.md (hunk 1 above)
Comment (janvanmansum) [reply README.md:42]: We will make a decision about this later. This PR doesn't have to wait for that.

**Comment by janvanmansum** [comment]
@rvanheest please, merge this if you are satisfied.
